lib/ansible/modules/network/meraki/meraki_network.py

```diff
@@ -172,7 +201,8 @@ def main():
     argument_spec = meraki_argument_spec()
     argument_spec.update(
         net_id=dict(type='str'),
-        type=dict(type='str', choices=['wireless', 'switch', 'appliance', 'combined'], aliases=['net_type']),
+        type=dict(type='str', choices=['wireless', 'switch', 'appliance'], aliases=['net_type']),
+        types=dict(type='list', aliases=['net_types']),
         tags=dict(type='str'),
```

Comment:
So this ought to be a list.

Reply:
I'm trying to work this out and can't figure how to do this without breaking backward compatibility. If I change `type` to a list it won't accept the strings as I've had it.

Reply:
Yes it will, it supports comma-separated values out of the box.

Reply:
Got it now.